.changeset/sharp-houses-study.md

```diff
@@ -0,0 +1,5 @@
+---
+'@clerk/clerk-js': patch
+---
+
+Improve UX for organization invitation form
```

Comment:
Let's add an empty changeset for this one as this message is rather abstract. 
Sorry for being so annoying the past few days with these things, but please bear with me as we're trying to set some guidelines!